flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
Why not build from source?

Reply:
otherwise this should be "only-arches" since you are not merely excluding aarch64 but actually acknowledging one arch exist.

Reply:
Good idea. I will fix this.

Reply:
> Why not build from source?

It seemed easier to just download the binary from the GitHub repo instead of making the build script download GoLang and compile it from source. Perhaps building from source would be better. I will update the manifest now.

Reply:
We'd prefer if you can build from source https://docs.flathub.org/docs/for-app-authors/requirements#best-practices

Reply:
done.